### PR TITLE
Feature/at 5908 sonar issues clean up

### PR DESCRIPTION
**Pull request by anhtran-ccpo**

### CODE REVIEW

File: atat/routes/portfolios/invitations.py [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 from .blueprint import portfolios_bp
 
+_PORTFOLIO_ADMIN_URL = "portfolios.admin"
```
[the remark this answers is not in the document]
Comment (anhtran-ccpo) [reply atat/routes/portfolios/invitations.py:17]: Yes. This change was due to duplicate usage of the string "portfolios.admin" so a variable is in place of all those duplicate usages

File: atat/domain/csp/cloud/mock_cloud_provider.py [hunk 2]
```diff
@@ -86,6 +85,8 @@ class MockCloudProvider(CloudProviderInterface):
     ATAT_ADMIN_CREATE_FAILURE_PCT = 12
     UNAUTHORIZED_RATE = 2
 
+    _ZOMBO_URL = "https://zombo.com"
```
[the remark this answers is not in the document]
Comment (anhtran-ccpo) [reply atat/domain/csp/cloud/mock_cloud_provider.py:88]: Yes. Done

**Review by kylelaker-ccpo — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # Node modules get installed here
 node_modules/
 
+.idea
```
Comment (jasonburkert-ccpo) [inline .gitignore:4]: Is the intent for here to ignore IntelliJ Idea?  If so, suggest moving this near the VSCode entry near the bottom and adding comment as to what it's for

Comment (anhtran-ccpo) [reply .gitignore]: Done

File: atat/routes/dev.py [hunk 4]
```diff
@@ -50,7 +52,7 @@ def random_service_branch():
         "email": "sam@example.com",
         "service_branch": random_service_branch(),
         "phone_number": "1234567890",
-        "citizenship": "United States",
+        "citizenship": _US,
         "designation": "military",
```
Comment (jasonburkert-ccpo) [inline atat/routes/dev.py:56]: Suggest treating string "military" in the same manner as "United States".  Seems to be used the same way for each dev user.

Comment (anhtran-ccpo) [reply atat/routes/dev.py]: Added variable for military and the repeated phone number

File: script/add_users_to_aad_tenant.py [hunk 5]
```diff
@@ -56,20 +54,20 @@ def create_user(token, tenant_id, tenant_host_name):
             "first_name": first_name,
             "last_name": last_name,
             "login_name": payload.user_principal_name,
-            "dod_id": "".join(random.choices(string.digits, k=10)),
+            "dod_id": "".join(choice(string.digits) for _ in range(10)),
```
Comment (jasonburkert-ccpo) [inline script/add_users_to_aad_tenant.py:57]: Thumbs up.
>>> "".join(choice(string.digits) for _ in range(10))
'2442851449'
>>> "".join(choice(string.digits) for _ in range(10))
'1813967221'
>>> "".join(choice(string.digits) for _ in range(10))
'1178387466'
>>> "".join(choice(string.digits) for _ in range(10))
'0545303154'
>>> "".join(choice(string.digits) for _ in range(10))
'4506780940'
>>> "".join(choice(string.digits) for _ in range(10))
'5264780591'
>>> "".join(choice(string.digits) for _ in range(10))
'7405759417'
>>> "".join(choice(string.digits) for _ in range(10))
'5162378884'
>>> "".join(choice(string.digits) for _ in range(10))
'5232860042'
>>> "".join(choice(string.digits) for _ in range(10))
'9489312443'
>>> "".join(choice(string.digits) for _ in range(10))
'9900485234'
>>> "".join(choice(string.digits) for _ in range(10))
'6364850691'
>>> "".join(choice(string.digits) for _ in range(10))
'7106420204'
>>> "".join(choice(string.digits) for _ in range(10))

**Review by jasonburkert-ccpo — APPROVED** [review]

**Review by jasonburkert-ccpo — APPROVED** [review]

**Review by jasonburkert-ccpo — APPROVED** [review]

**Review by jasonburkert-ccpo — APPROVED** [review]

File: tests/routes/test_dev.py [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+
+def test_users():
+    """
+    Validates users in _DEV_USERS
+    """
+    names = [user["first_name"] for user in _DEV_USERS.values()]
+    for user in _DEV_USERS.values():
+        assert user["citizenship"] == "United States"
+        assert user["first_name"] in names
```
Comment (jasonburkert-ccpo) [inline tests/routes/test_dev.py:11]: Great idea.

**Review by jasonburkert-ccpo — APPROVED** [review]

File: tests/routes/test_dev.py (hunk 6 above)
Comment (anhtran-ccpo) [reply tests/routes/test_dev.py:11]: Yeah i hope it works. The analysis didnt pick it up though for some reason. I just reran the job again hoping it picks it up

**Pull request merged**